m4_sessions/1706-inning/group_two.md

```diff
@@ -13,27 +17,35 @@ Students should be able to do the following:
-  * Mark to introduce topic and what it is (3 mins)
-  * Stats!?
-  * Watch clips from 60 mins 
-  * Podcast clips 
-* Group Discussion Time (20 mins)
-  * Group students into 5 groups (max) and assign each a company - ask them to do research on the company’s mission and discuss first in your small group how effective they’ve been in meeting their mission, how are they contributing to the attention economy
-  * 5 Companies:
-    * Facebook
-    * Snapchat
-    * Meditation Company?? 
-    * Google??
-    * Amazon
-* Whole Group Discussion (20 mins)
-  * Have each group present some of their findings (2 mins each)
-  * Are there any benefits to the “Attention Economy”.
-  * What could Google do as a company, or us as developers do to change the attention economy landscape? 
+  * Briefly define “Attention Economy”, as the current state of the scarce resource of “Attention” in a world of nearly infinite information.
+  * When people are asked if they think their time spent on most of these types of applications, over 90% regret how they spent their time. Present self (in the moment, using application) vs. Future Self (with feelings of regret)
+  * The reason these technologies are so successful:
+   * Take advantage of our psychology 
+   * Machine learning identifies what we like (even if it’s things that create outrage, actually especially outrage) 
+   * Tristian Harris - technology ethicist (inspiration of this gear up) took a class in “Persuasive Technology” during his time at Stanford
+   * Some of his classmates went on to create Instragram and LinkedIn
+   * However, there was never a discussion about if taking advantage of the human psyche in this manner is ethical
+   * Watch clips from Youtube: https://www.youtube.com/watch?v=50R21mblLb0 
+
+* Small Group Discussion Time (15 mins)
+  * What are some other examples of how you’re susceptible to the ‘attention economy’ that you’ve noticed?  [Dave]
+  * Are there any benefits to the “Attention Economy”? [Brett]
+  * Whose responsibility is it to spend our time wisely (technology companies or us as human)? [Craig]  
+  * If it’s up to the company, what are some strategies they could employ to ensure that their users are spending their time well while using their applications, while also maintaining revenue? 
+  * If the responsibility is us on as humans to avoid clicking and using, what could we do to spend our time wisely, while also not exhibiting too much “FOMO”? 
+
+* Whole Group Discussion (25 mins)
+ * Have each group present some of their findings (2 mins each)
+ * Let’s say you have a meditation application that you know will help people, but the only way compete with Facebook, Snapchat, etc. is to add daily notifications and maybe a meditation streak game, which kind of defeats the purpose of an app that’s supposed to bring you emotional peace. 
+   * As a junior, what do you do? 
+   * As a more senior “technologist” (senior dev, or founder / CTO)
+
```

Comment:
I would add some sort of "closing" even if it is final thoughts from each of you sharing your opinions/answer to the question.